packages/legend-graph/src/graph-manager/action/changeDetection/DSL_Service_ObserverHelper.ts

```diff
@@ -224,6 +229,37 @@ export const observe_MultiExecutionTest = skipObserved(
   },
 );
 
+export const observe_Ownership = (
```

Comment:
this function needs to be defined after observe_deploymentOwnership and observe_userListOwnership